### DISCUSSION TITLE
New Release Does Not Compile

### DISCUSSION THREAD

**Discussion by Cal-LF** [mriscoc/Ender3V2S1#76] [General]
@mriscoc Using repo code download, manual mesh v427, does not compile.

**Comment by mriscoc** [comment]
Fixed, thanks.

**Reply by carioca53** [reply]
Same problem with 422-BLTouch, does not compile

**Reply by carioca53** [reply]
It is missing "proui" lib folder

**Reply by mriscoc** [reply]
Was already fixed, re-download from the repository.

**Reply by carioca53** [reply]
ok Thank you

**Comment by mriscoc** [comment]
@Cal-LF I write this hoping you're okay.
